.github/workflows/test.yml

```diff
@@ -14,13 +14,25 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup
+      - uses: actions/cache@v3
```

Comment:
Could you add that blog post as a comment in this file somewhere? Useful as a reference if someone needs to touch these